README.md

```diff
@@ -19,6 +19,7 @@
 
 * ImageMagick or GraphicsMagick (for MiniMagick);
 * [Bootstrap 3](http://getbootstrap.com/) fully installed in your app.
+* jQuery
```

Comment:
"Bootstrap 3 fully installed in your app" already implies that jQuery is required.